filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -328,6 +329,8 @@ json.add_error_key: true
 and its value must be string, otherwise it is ignored. If no text key is defined, the line
 filtering and multiline features cannot be used.
 
+*`json_key`*:: name of the JSON key in the output document where the decoded JSON has to be placed. The default is "json"
```

Comment:
The name stutters in the config as it is `json.json_key`. I think it should be called `target` which is more consistent with Logstash's [json filter](https://www.elastic.co/guide/en/logstash/current/plugins-filters-json.html#plugins-filters-json-target).